exporter/cassandraexporter/README.md

```diff
@@ -18,7 +18,8 @@ The following settings can be optionally configured:
 
 - `dsn` The Cassandra server DSN (Data Source Name), for example `127.0.0.1`.
   reference: [https://pkg.go.dev/github.com/gocql/gocql](https://pkg.go.dev/github.com/gocql/gocql)
-- `port` (default = 9042) The Cassandra server Port 
+- `port` (default = 9042) The Cassandra server Port
+- `timeout` (default = 3 second) The Cassandra server connection timeout (Second)
```

Comment:
```suggestion
- `timeout` (default = 3s) The Cassandra server connection timeout
```

Nit: Change style to match [other timeout definitions](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/exporterhelper#configuration).